test/TestImport.hs

```diff
@@ -58,16 +64,35 @@ getTables = do
 
     return $ map unSingle tables
 
+
+bodyNotContains :: String -> YesodExample site ()
```

Comment:
I've created a PR to move this assert into Yesod core - https://github.com/yesodweb/yesod/pull/1271